dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -147,42 +160,60 @@ export default defineComponent({
       }
     },
     /**
-     * Handles selection of a file by the user. If duplicates are found in the selection, this is handled.
+     * Handles selection of files by the user. If invalid file names are found in the selection, this is handled.
      * At the end an event is emitted reflecting that the referenceable reports have updated.
-     * @param documents the list of all documents for the upload, determined by the selection in the file uploader
+     * @param selectedDocumentsForUpload the list of all selectedDocumentsForUpload for the upload, determined by the selection in the file uploader
      */
-    handleUpdatedDocumentsSelectedForUpload(documents: DocumentToUpload[]) {
-      this.documentsToUpload = documents;
-      const duplicatesWithIndex: DuplicateWithIndex[] = [];
+    handleUpdatedDocumentsSelectedForUpload(selectedDocumentsForUpload: DocumentToUpload[]) {
+      this.documentsToUpload = selectedDocumentsForUpload;
 
-      if (this.areDuplicatesAmongReferenceableReportNames()) {
-        const foundExistingRecords = new Set<string>();
+      const invalidFileNamesWithIndexAndReason: InvalidFileNameWithIndexAndReason[] = [];
```

Comment:
`invalidFileNamesWithIndexAndReason` is a bit misleading that this might only contain files with invalid filenames. What about `nameIndexAndReasonOfInvalidFiles` or `invalidFilesByNameIndexAndReason`?

Reply:
I renamed the variables and the type to "NameIndexAndReasonOfInvalidFile(s)"

Reply:
👍